README.md

```diff
@@ -283,6 +283,27 @@ scss-lint --format=XML [scss-files...]
 </lint>
 ```
 
+### CheckStyle
+
+Outputs a CheckStyle XML with `<checkstyle>`, `<file>`, and `<error>` tags. Suitable for
+consumption by tools like [Jenkins](http://jenkins-ci.org/) with [CheckStyle plugin](https://wiki.jenkins-ci.org/display/JENKINS/Checkstyle+Plugin).
+
+```bash
+scss-lint --format=CheckStyle [scss-files...]
+```
+
+```xml
+<?xml version="1.0" encoding="utf-8"?>
+<checkstyle version="1.5.6">
```

Comment:
The version number you've specified seems really out of date. The current version of the CheckStyle plugin is 3.41.

I guess my issue with this is that this plugin is extremely specific to a particular testing platform (Jenkins) and isn't a generic output format (e.g. JSON).

I would rather this be pulled into its own separate gem and loaded via the `--require` and `--format` flags. I don't think this formatter should be shipped with `scss-lint`.